src/views/ContinuationIn/ContinuationInAuthorization.vue

```diff
@@ -1,8 +1,11 @@
 <template>
   <div id="continuation-in-authorization">
     <!-- Name -->
-    <section class="mt-10">
-      <header id="name-header">
+    <section
+      id="name-section"
```

Comment:
The id is used for scrolling to the section, so I think it makes more sense to scroll to the section element instead of the header element.